routers/payments.py

```diff
@@ -0,0 +1,52 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/payment")
```

Comment:
```suggestion
@router.get("/payments")
```
Don't understand a set of arguments, what payments do you expect to get here

Reply:
I use them for filtering: Booking id, amount of money, time of payment